config/txgh.yml

```diff
@@ -4,12 +4,12 @@ txgh:
             matthewjackowski/txgh-test-resources:
                 api_username: <%= ENV['GITHUB_USERNAME']  %>
                 api_token: <%= ENV['GITHUB_TOKEN'] %>
-                push_source_to: <%= ENV['GITHUB_PUSH_SOURCE_TO'] %>
-                branch: <%= ENV['GITHUB_BRANCH'] %>
+                push_source_to: txgh-test-1
```

Comment:
Oh ok sure. I didn't understand why they were templated. Can't you configure txgh to work with more than one repository?